### ISSUE TITLE
Jupyterlab 1.1.0 not compatible with Python 3.5 due to f-string usage?

### ISSUE THREAD

**Issue by dotsdl** [jupyterlab/jupyterlab#7124]
## Description

Jupyterlab 1.1.0 hits a `SyntaxError` when run on Python 3.5.2 (Ubuntu 16.04 Python 3 via apt):

```
Traceback (most recent call last):
  File "/usr/local/bin/jupyter-lab", line 6, in <module>
    from jupyterlab.labapp import main
  File "/usr/local/lib/python3.5/dist-packages/jupyterlab/labapp.py", line 22, in <module>
    from .commands import (
  File "/usr/local/lib/python3.5/dist-packages/jupyterlab/commands.py", line 557
    self.logger.info(f'Building jupyterlab assets ({command})')
                                                             ^
SyntaxError: invalid syntax
```
This appears to be due to the use of an f-string in one of the modules, which will only work in Python 3.6+: [commands.py](https://github.com/jupyterlab/jupyterlab/blame/v1.1.0/jupyterlab/commands.py#L557). According to the [changelog](https://jupyterlab.readthedocs.io/en/stable/getting_started/changelog.html#v1-1-0), JupyterLab v1.1.0 should still support 3.5+.

## Context

- Operating System and version: Ubuntu 16.04 LTS
- Browser and version: Firefox 68.0.2
- JupyterLab version: 1.1.0

**Comment by jasongrout** [comment]
Or even more directly, setup.py and the conda-forge package indicates python >=3.5: 

https://github.com/jupyterlab/jupyterlab/blob/master/setup.py#L28

https://github.com/conda-forge/jupyterlab-feedstock/blob/3d04ed845b32a57705d193278ea928771af41e44/recipe/meta.yaml#L30

**Comment by jasongrout** [comment]
Fix is in jlab 1.1.1, just released on pypi and in process of going up on conda-forge
